src/app/components/cards/MarkdownViewer.jsx

```diff
@@ -199,9 +213,8 @@ class MarkdownViewer extends Component {
                 } else if (type === 'dtube') {
                     const url = `https://emb.d.tube/#!/${id}`;
                     sections.push(
-                        <div className="videoWrapper">
+                        <div className="videoWrapper" key={idx++}>
                             <iframe
```

Comment:
Please use something other than index for the key as this will confuse React's Dom diffing, when for example more than one element has a key of say: 1...
The key ought to be a globally (to the dom) value... in this case url would likely suffice.